src/sentry/wsgi.py

```diff
@@ -16,3 +17,16 @@
 
 # Run WSGI handler for the application
 application = WSGIHandler()
+
+# trigger a warmup of the application
+application(
+    {
+        "PATH_INFO": "/_health/",
```

Comment:
ideally this would be a specific `/_warmup` view or something (which at minimum gets ~4 more modules warmed) but this is a good start at least.  with a dedicated warmup view we could probably tune this better (and do more stuff like set up memcached connections / whatever else)

Reply:
fwiw these are the 4 additional modules that get imported by running a "real" view (crossdomain.xml):
```diff
$ diff -u <(jq . mods2.json) <(jq . mods.json)
--- /dev/fd/63	2024-09-13 16:41:26
+++ /dev/fd/62	2024-09-13 16:41:26
@@ -655,6 +655,9 @@
   "django.contrib.messages.middleware",
   "django.contrib.messages.storage",
   "django.contrib.messages.storage.base",
+  "django.contrib.messages.storage.cookie",
+  "django.contrib.messages.storage.fallback",
+  "django.contrib.messages.storage.session",
   "django.contrib.messages.utils",
   "django.contrib.postgres",
   "django.contrib.postgres.constraints",
@@ -682,6 +685,7 @@
   "django.contrib.sessions.exceptions",
   "django.contrib.sessions.middleware",
   "django.contrib.sessions.models",
+  "django.contrib.sessions.serializers",
   "django.contrib.sites",
   "django.contrib.sites.apps",
   "django.contrib.sites.checks",
```